### PR TITLE
To Be Review the part

### PR DESCRIPTION
**Pull request by Kinfe123**
what do you think about the solution?

### CODE REVIEW

File: 0025-reverse-nodes-in-k-group/0025-reverse-nodes-in-k-group.py [hunk 1]
```diff
@@ -10,11 +10,14 @@ def getTheKth(self, nodes , k):
...
         dummy = ListNode()
         dummy.next = head
         prevOnes = dummy
         #NOw dummy and previous one is pointing to the same head 
-    
+        
+        
         #Start catching the end of the group and reversing the group 
         while True:
             endOfGroup = self.getTheKth(prevOnes , k)
```
Comment (million-t) [inline 0025-reverse-nodes-in-k-group/0025-reverse-nodes-in-k-group.py:23]: try adding line breaks between the if and while blocks - makes it more readable.

**Review by million-t — COMMENTED** [review]
Over all you wrote a good code. Some qualities I observed, to mention, are modularizing, commenting blocks, descriptive variable naming etc. For faster comprehension, though, try adding line gaps between code blocks

File: 0025-reverse-nodes-in-k-group/0025-reverse-nodes-in-k-group.py (hunk 1 above)
Comment (Kinfe123) [reply 0025-reverse-nodes-in-k-group/0025-reverse-nodes-in-k-group.py:23]: Noticed

**Comment by Kinfe123** [comment]
Noticed , Patched

**Pull request merged**